src/security/cli_security.c

```diff
@@ -67,7 +66,12 @@ request_credentials_via_drpc(Drpc__Response **response)
 	struct drpc	*agent_socket;
 	int		rc;
 
-	agent_socket = drpc_connect(get_agent_socket_path());
+	if (dc_agent_sockpath == NULL) {
+		D_ERROR("Unable to craft DRPC socket path\n");
+		return -DER_BADPATH;
```

Comment:
There is a -DER_UNINIT - I'd say this case means the agent path wasn't initialized. Would be nice to make the log message more specific too

Reply:
Didn't realize that one existed. That is a much better code. Also changed the error message to better reflect the error.